sources/packages/backend/libs/sims-db/src/entities/application-offering-change-request.model.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryColumn,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { Application } from "./application.model";
+import { EducationProgramOffering } from "./education-program-offering.model";
+import { User } from "./user.model";
+import { Note } from "./note.model";
+import { ApplicationOfferingChangeRequestStatus } from "./application-offering-change-request-status.type";
+
+const REASON_MAX_LENGTH = 500;
```

Comment:
This max length could be a const like NOTES_MAX_LENGTH which could be used on POST API for class validators.

Reply:
I didn't get the point here, i believe it should done in the submit story

Reply:
I meant this constant must be more global as we see that it has more potential usage. like this.

![image](https://github.com/bcgov/SIMS/assets/54600590/8539dff1-ae23-4e68-a50d-fe80afe7f663)

Reply:
![image](https://github.com/bcgov/SIMS/assets/77353155/31a2ef60-1ff4-4c67-88d8-31985c6f9595)
I believe we can add export when we are actually using it

Reply:
Yes just the export is missing 😊